task/tkn-bundle/0.1/tkn-bundle.yaml

```diff
@@ -96,15 +100,42 @@ spec:
           done
         fi
       done
+      for f in "${FILES[@]}"; do
+        if [[ -n "$(params.STEPS_IMAGE)" ]]; then
```

Comment:
If it's not set, maybe don't even enter the for-loop?

Reply:
Cleaned it up, should be easier to understand.